step-framework-parent/step-framework-collections/src/test/java/step/core/collections/inmemory/InMemoryCollectionTest.java

```diff
@@ -9,4 +9,9 @@ public class InMemoryCollectionTest extends AbstractCollectionTest {
     public InMemoryCollectionTest() {
         super(new InMemoryCollectionFactory(new Properties()));
     }
+
+    @Override
+    public void renameCollections() {
+        //not implemented for in memory

```

Comment:
Actually it should work too. Here I would also check that the value of the field "name" has been changed